### ISSUE TITLE
can't get kr rankings.

### ISSUE THREAD

**Issue by berrybar** [facundoolano/google-play-scraper#495]
* Operating System: windows 10
* Node version: 10.16.0
* google-play-scraper version: 8.0.4

### Description: 
When I tested in the US and Japan, I was able to get the value.
But when I set it to Korea, I couldn't get the value at all.
It hasn't been since December 1st in Korea, and it seems to have found a problem.
It seems that the Korean Google Play ranking collection url is pointing to the wrong path.
Can I get feedback on this?

### Example code:
```js
await G.list({
            category: G.category.GAME,
            collection: collection,
            num: 200,
            lang: 'ko',
            country: 'kr',
            throttle: 10
        })
```

### Error message:

```
Error: The collection topgrossing is invalid for the given category, top apps or new apps
```

**Comment by ansoncen** [comment]
What I found in the region kr, hk and tw, app detail page layout upgrade to the new version.  The lib's parse function should be updated so.

**Comment by berrybar** [comment]
I agree with that opinion.

**Comment by druphliu** [comment]
@VitaliyRDev

**Comment by 13389203735** [comment]
> on 7 Dec 2021

I also encountered this problem. How did you solve it? Please tell me

**Comment by 13389203735** [comment]
> 我在kr、hk和tw地区发现，app详情页布局升级到新版本。lib的解析
我也出现了这个问题，你是怎么解决的，麻烦告知一下

**Comment by 13389203735** [comment]
> What I found in the region kr, hk and tw, app detail page layout upgrade to the new version. The lib's parse function should be updated so.

I use the dev branch code locally, but I found that the app store can't get information in Kr, HK and tw

I also have this problem. How did you solve it? Please tell me

**Comment by 13389203735** [comment]
@ansoncen I use the dev branch code locally, but I found that the app store can't get information in Kr, HK and tw

I also have this problem. How did you solve it? Please tell me

**Comment by berrybar** [comment]
> > on 7 Dec 2021
> 
> I also encountered this problem. How did you solve it? Please tell me

I also haven't solved it yet.

**Comment by icarcal** [comment]
Looks like google play completely changed the kr store as @ansoncen pointed out.
The URL used by the lib is redirecting to `/apps`:
`https://play.google.com/store/apps/top/category/GAME/?hl=ko&gl=kr`

I'm not sure if this new store version has the `top` or `new` URL anymore.

Apparently, it does have some way to get the top/new apps, but differently than we do today:
`https://play.google.com/store/apps/category/GAME?hl=ko&gl=kr`

**Comment by 13389203735** [comment]
> @icarcal 

There is still no way to solve this problem, right? Now apps in Kr and other countries can't get useful information

**Comment by ansoncen** [comment]
> Looks like google completely changed the kr store as @ansoncen pointed out. The URL used by the lib is redirecting to `/apps`: `https://play.google.com/store/apps/top/category/GAME/?hl=ko&gl=kr`
> 
> I'm not sure if this new store version has the `top` or `new` URL anymore.
> 
> Apparently, it does have some way to get the top/new apps, but differently than we do today: `https://play.google.com/store/apps/category/GAME?hl=ko&gl=kr`

I have raised a concern that google may sooner or later apply the new store to all the regions.

**Comment by facundoolano** [comment]
Yes i think that's very likely. Ideally we'd have support for both and switch when they do, but I don't have the bandwidth to start working on the new scraper at the moment.

**Comment by 89z** [comment]
If you only need the Korean language, I have a solution. If it needs to be specific to South Korea, I cannot help.

**Comment by ansoncen** [comment]
Any updates on this? I found all the play store page has been updated to the new layout, while the API still works for the time. I found some browsers still can open the old version (could be the A/B test), this might be the reason the API still works for now.

**Comment by facundoolano** [comment]
can you check if this still happens with the latest version?

**Comment by facundoolano** [comment]
Issue automatically closed. Please create a new one if the issue persists
